apps/sms/test/marionette/lib/messages.js

```diff
@@ -112,6 +112,11 @@ var ParticipantsAccessor = require('./views/participants/accessors');
           newMessageView.addNewRecipient(recipient);
         },
 
+        getActivePanelName: function() {
+          var cls = client.findElement('.panel-active').getAttribute('class');
+          return cls.match(/(panel-[A-Z][a-zA-Z]+)/g);
```

Comment:
This is working by chance: `match` returns an array, but the conversation array -> string returns the correct value in the assertion.

I think we should return something more expected here.

You can do this:
```javascript
var matchResult = /\bpanel-\w+\b/g.exec(cls);
return matchResult && matchResult[0];
```

or even:
```javascript
var matchResult = /\bpanel-(\w+)\b/g.exec(cls);
return matchResult && matchResult[1];
```
=> and you get `ConversationView` instead of `panel-ConversationView`. What do you think ?